js/setup.js

```diff
@@ -61,3 +61,80 @@ var getFragment = function () {
 getFragment();
 
 userDialog.querySelector('.setup-similar').classList.remove('hidden');
+
+var setupOpen = document.querySelector('.setup-open');
+var setupClouse = userDialog.querySelector('.setup-close');
+var userNameInput = userDialog.querySelector('.setup-user-name');
+var button = document.querySelector('.button');
+
+var onPopupEscPress = function (evt) {
+  if (evt.keyCode === 27) {
+    closePopup();
+  }
+};
+
+var openPopup = function () {
+  userDialog.classList.remove('hidden');
+  document.addEventListener('keydown', onPopupEscPress);
```

Comment:
```suggestion
  document.addEventListener('keydown', onDocumentKeydown);
```
https://up.htmlacademy.ru/javascript/19/criteries#d4
> Из названия обработчика события и функции-коллбэка следует, что это обработчик.

> on + (на каком элементе) + что случилось

Легко составлять названия функций обработчиц событий: как видишь слова eventListener - значит это обработчица события. Ты выбрала приставку on + потом то что впереди до точки - это элемент, на котором мы ловим собитие + то что в кавычках после - это само название события, какое мы ловим.

Reply:
в данном случае, это блок кода из демки. onPopupEscPress - это функция, которая содержит в себе кнопку esc, я не понимаю, зачем её переименовывать, если её название соответствует ее содержанию?

Reply:
Её название не соответствует критерию написания функций обработчиков, поэтому её надо переделать